package.json

```diff
@@ -34,14 +37,14 @@
         "build-all": "yarn build && yarn build-template && yarn build-website",
         "start": "yarn workspace babylonjs-editor start",
         "package": "yarn clean && yarn clean:node_modules && yarn && yarn lint && yarn build-all && yarn test && node build.mjs",
-        "clean": "rm -rf editor/build editor/declaration editor/electron-packages editor/coverage quixel/build quixel/declaration tools/build tools/declaration tools/coverage template/.next website/.next",
-        "clean:node_modules": "rm -rf ./node_modules ./editor/node_modules ./quixel/node_modules ./tools/node_modules ./template/node_modules ./website/node_modules",
+        "clean": "rimraf editor/build editor/declaration editor/electron-packages editor/coverage quixel/build quixel/declaration tools/build tools/declaration tools/coverage template/nextjs/.next template/solidjs/dist template/vanillajs/dist website/.next",
+        "clean:node_modules": "rimraf --glob **/node_modules",
         "test": "yarn workspace babylonjs-editor-tools test && yarn workspace babylonjs-editor test",
         "coverage": "yarn workspace babylonjs-editor-tools coverage && yarn workspace babylonjs-editor coverage",
         "lint-editor": "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --cache --no-warn-ignored ./editor/src/**",
         "lint-tools": "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --cache --no-warn-ignored ./tools/src/**",
         "lint-quixel": "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --cache --no-warn-ignored ./quixel/src/**",
-        "lint-template": "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --cache --no-warn-ignored ./template/src/**",
+        "lint-template": "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --cache --no-warn-ignored ./template/**/src/**",
```

Comment:
should be named "lint-templates" (with the s)

Reply:
see my comment above about folder name conflicts